security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,64 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+* With this release of Compliance Operator 1.7.0, BSI profile support is added for OCP. For more information, refer to the KCS article (link:https://access.redhat.com/articles/7045834[*BSI Quick Check*]) and (link:https://access.redhat.com/compliance/bsi[*BSI Compliance Summary*]).
+
+[id="compliance-operator-1-7-0-bug-fixes_{context}"]
+=== Bug fixes
+
+* Before this release, Compliance Operator would provide an unneeded remediation recommendation due to differences in filesystem structure for the `s390x` architecture. With this release, the Compliance Operator now recognizes the differences in filesystem structure and does not provide the misleading remediation. With this update, the rule is now more clearly defined. (link:https://issues.redhat.com/browse/OCPBUGS-33194[*OCPBUGS-33194*])
+
+* Previously, the instructions for rule `ocp4-etcd-unique-ca` did not work for OCP 4.17 and later. With this update, the instructions and actionable steps are corrected. (link:https://issues.redhat.com/browse/OCPBUGS-42350[*OCPBUGS-42350*])
+
+* When using the Compliance Operator with Cluster Logging Operator (CLO) version 6.0, various rules would fail. This is due to backwards incompatible changes to the CRDs that CLO uses. The Compliance Operator relies on those CRDs to verify logging functionality. The CRDs have been corrected to support the PCI-DSS profiles with CLO. (link:https://issues.redhat.com/browse/OCPBUGS-43229[*OCPBUGS-43229*])
+
+* After installing logging 6.0, users found that the ComplianceCheckResult `ocp4-cis-audit-log-forwarding-enabled` was failing because there was a change in the APIversion of `clusterlogforwarder`. Log collection and forwarding configurations are now specified under the new API, part of the observability.openshift.io API group. (link:https://issues.redhat.com/browse/OCPBUGS-43585[*OCPBUGS-43585*])
+
+* For prior releases of Compliance Operator, the URL in the OCP4 GUI console shows (https://docs.openshift.com/container-platform/4.10/security/compliance_operator/compliance-operator-understanding.html), which is incorrect for versions of OCP after version 4.10. As of OCP version 4.11 and later, the link includes the term "co-concepts", such as: https://docs.openshift.com/container-platform/4.11/security/compliance_operator/co-concepts/compliance-operator-understanding.html. As of this release, the URL shown in the OCP4 GUI has been corrected. (link:http://issues.redhat.com/browse/OCPBUGS-48090[*OCPBUGS-48090*])
+
+* For previous releases of Compliance Operator, the scans would generate an error log for the reconcile loop on the operator pod. With this release, the Compliance Operator controller logic is more stable and does not generate the errors for the reconcile loop. (link:https://issues.redhat.com/browse/OCPBUGS-51267[*OCPBUGS-51267*])
```

Comment:
"does not generate the errors for the reconcile loop" it is not accurate. The error frequency decreased a lot but it still  exists.